src/components/NewComps/FeedbackCards/index.js

```
@@ -0,0 +1,84 @@
+import { useEffect, useRef, useMemo } from "react";
+import styles from "./FeedbackCards.module.scss";
+import { MdOutlineFormatQuote } from "react-icons/md";
+
+export default function FeedbackCards({ testimonials }) {
+  if (testimonials?.data?.length === 0) return null;
+  const scrollRef = useRef(null);
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)